src/main/java/sirius/db/jdbc/SmartQuery.java

```diff
@@ -105,6 +113,91 @@ public SmartQuery<E> where(SQLConstraint constraint) {
         return this;
     }
 
+    /**
+     * Adds a "USE INDEX" hint for a table that recommends an index to te optimizer.
+     *
+     * @param table     the table for which the index should be used.
+     * @param indexName the name of the index to use.
+     * @param <T>       the type of the entities being queried.
+     * @return the query itself for fluent method calls.
+     */
+    public <T extends SQLEntity> SmartQuery<E> useIndex(Class<T> table, String indexName) {
+        return useIndex(table, null, indexName);
+    }
+
+    /**
+     * Adds a "USE INDEX" hint for a table that recommends an index to te optimizer.
+     *
+     * @param table     the table for which the index should be used.
+     * @param forHint   the action for which the index should be used. Can be null.
+     * @param indexName the name of the index to use.
+     * @param <T>       the type of the entities being queried.
+     * @return the query itself for fluent method calls.
+     */
+    public <T extends SQLEntity> SmartQuery<E> useIndex(Class<T> table, IndexForHint forHint, String indexName) {
+        indexHint(table, "USE", forHint, indexName);
+        return this;
+    }
+
+    /**
+     * Adds a "FORCE INDEX" hint for a table that forces an index to te optimizer.
```

Comment:
```suggestion
     * Adds a "FORCE INDEX" hint for a table that forces an index to the optimizer.
```
Is your IntellIJ spell checker disabled? 😇

Reply:
Looks like it 😲